preview/test-files/cve.payload.json

```diff
@@ -20,8 +22,11 @@
                 }
               ],
               "cve_count": 2
-            },
+            }
+          ],
+          "transitive_updates": [
             {
+              "is_transitive":true,
```

Comment:
When we are having 2 sections for direct and transitive, do we still need is_transitive field?

Reply:
In the backend code,I am using this flag to check if the dependency falls into transitive or direct category. Hence appending the dep dictionary into corresponding lists. Each dep dictionary by default contains this flag.